radar-upload-frontend/src/components/Common/UploadForm.vue

```diff
@@ -110,6 +110,7 @@ export default {
             throw new Error();
           });
         this.$emit('finishUpload', uploadingFile);
+        await fileAPI.markRecord({ recordId: returnedRecord.id, revision: returnedRecord.revision });
```

Comment:
One small change: Please use the values of `uploadingFile` instead of `returnedRecord`

Reply:
I don't think uploadingFile returns json though, it is an 204 No Content right? I think the current behavior is fine then.

Reply:
I just had a look. My bad. It does return a JSON, though it is not the metadata. I assumed we would increment the version when we do a upload the content. But we don't. So current behavior looks fine.